models/public/deep-speech-v0.5.0/deep-speech-v0.5.0.md

```diff
@@ -0,0 +1,13 @@
+# deep-speech-v0.5.0
```

Comment:
Please, provide description for input and output in format like [here](https://github.com/opencv/open_model_zoo/blob/develop/models/public/mask_rcnn_resnet101_atrous_coco/mask_rcnn_resnet101_atrous_coco.md).

Reply:
please check the latest commit for the information of deep speech model.